jetty-util/src/main/java/org/eclipse/jetty/util/RolloverFileOutputStream.java

```diff
@@ -176,11 +178,7 @@ public RolloverFileOutputStream(String filename,
         // Calculate Today's Midnight, based on Configured TimeZone (will be in past, even if by a few milliseconds)
         setFile(now);
 
-        synchronized (RolloverFileOutputStream.class)
-        {
-            if (__rollover == null)
-                __rollover = new Timer(RolloverFileOutputStream.class.getName(), true);
-        }
+        __rollover = new Timer(RolloverFileOutputStream.class.getName(), true);
```

Comment:
This will create a new static Timer for every instance. It will replace the previous instance, but it will be held in memory by the thread.
Either keep code as is, or make it statically initialised.